src/state/patients/reducer.ts

```diff
@@ -75,6 +76,13 @@ export default produce((draft: IPatientsState, action: IAction<any, any>) => {
       break;
     }
 
+    case SEARCH_PATIENT_RESET: {
+      draft.searchResults.status = "IDLE";
+      delete draft.searchResults.error;
+      delete draft.searchResults.data;
```

Comment:
It would be safer to restore initial state, which is 
`draft.searchResults.data = []`